### PR TITLE
Add a few programs and games

### PR DESCRIPTION
**Pull request by amir16yp**
Adds the following:

- Extreme Tux Racer 0.5 beta
- SuperTuxKart 1.3
- Mari0
- Ant Renamer
- Snappy Driver Installer Lite
all were tested on ReactOS 0.4.14 by me and members of the reactos discord

### CODE REVIEW

File: sdi-tool.txt [hunk 1]
```diff
@@ -0,0 +1,10 @@
+[Section]
+Name = Snappy Driver Installer Lite
+Version = 1.22.1
+License = Freeware
+Description = A helper tool for installing drivers.
+Category = 13
+URLSite = https://sdi-tool.org
+URLDownload = http://sdi-tool.org/releases/SDI_R2201.zip
```
Comment (JoachimHenze) [inline sdi-tool.txt:8]: Why not https in the URLDownload?

File: ant-renamer.txt [hunk 2]
```diff
@@ -0,0 +1,10 @@
+[Section]
+Name = Ant Renamer
+Version = 2.12.0
+License = GPLv2
+Description = Mass file renaming utility, allows for renaming of many files at once.
+Category = 12
+URLSite = https://www.antp.be/software/renamer
+URLDownload = http://update.antp.be/renamer/antrenamer2_install.exe
```
Comment (JoachimHenze) [inline ant-renamer.txt:8]: Why not https here?

**Comment by HBelusca** [comment]
Do we require screenshots proving that those programs and games indeed work in ReactOS?

**Comment by amir16yp** [comment]
> Do we require screenshots proving that those programs and games indeed work in ReactOS?

me and a person named winterhell on the ros discord tested STK and extreme tux racer. the newest version of extreme tux racer that works on ros is 0.5 beta and newer versions install but get unimplemented api call errors when launched, and for some reason cant be uninstalled. I tested mari0 myself however i couldn't verify if sound works (i was using real hardware and sound was not working). Snappy Driver Installer is a program that is recommended by the ROS wiki and i was surprised it wasn't already in rapps. Ant Renamer is a rather simple program and i tested it in ros by renaming a bunch of mp3 files

**Comment by learn-more** [comment]
> > Do we require screenshots proving that those programs and games indeed work in ReactOS?
> 
> me and a person named winterhell on the ros discord tested STK and extreme tux racer. the newest version of extreme tux racer that works on ros is 0.5 beta and newer versions install but get unimplemented api call errors when launched, and for some reason cant be uninstalled. I tested mari0 myself however i couldn't verify if sound works (i was using real hardware and sound was not working). Snappy Driver Installer is a program that is recommended by the ROS wiki and i was surprised it wasn't already in rapps. Ant Renamer is a rather simple program and i tested it in ros by renaming a bunch of mp3 files

Did you test snappy driver installer?
There are some bug reports of it not working.

**Comment by amir16yp** [comment]
> > > Do we require screenshots proving that those programs and games indeed work in ReactOS?
> > 
> > 
> > me and a person named winterhell on the ros discord tested STK and extreme tux racer. the newest version of extreme tux racer that works on ros is 0.5 beta and newer versions install but get unimplemented api call errors when launched, and for some reason cant be uninstalled. I tested mari0 myself however i couldn't verify if sound works (i was using real hardware and sound was not working). Snappy Driver Installer is a program that is recommended by the ROS wiki and i was surprised it wasn't already in rapps. Ant Renamer is a rather simple program and i tested it in ros by renaming a bunch of mp3 files
> 
> Did you test snappy driver installer? There are some bug reports of it not working.

i did test snappy driver installer on 0.4.14, it worked alright

**Review by learn-more — APPROVED** [review]

**Comment by JoachimHenze** [comment]
Yes I bet you could start up Snappy Driver installer.
Could you actually download a driver with Snappy Driver Installer? I guess that wouldn't work yet.
But for me this is no blocker, I just wanted to mention it for the sake of avoiding misconceptions.

**Comment by learn-more** [comment]
Thanks for your contribution, sorry for the delay in merging.

**Pull request merged**